specs/abtesting-v3/common/schemas/Variant.yml

```
@@ -0,0 +1,94 @@
+variants:
+  type: array
+  description: |
+    A/B test variants.
+
+    The first variant is your _control_ index, typically your production index.
+    All of the additionnal variants are indexes with changed settings that you want to test against the control.
+  items:
+    $ref: "#/variant"
+
+variant:
+  type: object
+  additionalProperties: false
+  properties:
+    description:
+      $ref: "../parameters.yml#/description"
+    estimatedSampleSize:
+      type: integer
+      description: |
+        Estimated number of searches required to achieve the desired statistical significance.
+
+        The A/B test configuration must include a `mininmumDetectableEffect` setting for this number to be included in the response.
+      example: 0
+    filterEffects:
+      $ref: "../parameters.yml#/filterEffects"
+    index:
+      $ref: "../parameters.yml#/index"
+    trafficPercentage:
+      $ref: "../parameters.yml#/trafficPercentage"
+    metrics:
+      $ref: "#/metrics"
+  required:
+    - index
+    - description
+    - trafficPercentage
+    - metrics
+
+metrics:
+  type: array
+  description: All ABTest metrics that were defined during test creation
+  items:
+    $ref: "#/metric"
+
+metric:
+  type: object
+  properties:
+    name:
+      type: string
+    updatedAt:
+      type: string
+      description: Date and time when the metric was last updated, in RFC 3339 format.
+    value:
+      type: number
+      format: double
+    pvalue:
+      type: number
+      format: double
+    dimension:
+      type: string
+      description: Dimension defined during test creation
+    winsorizedValue:
+      type: number
+      format: double
+      description: |
+        Only present in case the metric is a revenue. 
+        It is the normalized value of it. 
+        It is tied to a per revenue-currency pair contrary to other 
+        global filter effects (such as outliers and empty search count).
+  required:
+    - name
+    - updatedAt
+    - value
+    - pvalue
+  example:
+    - name: addToCartCount
+      updatedAt: 2025-06-15T15:06:44.400601Z
+      value: 5
+      pvalue: 0.01
```

Maybe we can add a description here (or somewhere) indicating that the pValue for the first variant (control) will always be 0. And that we don't do pairwise comparisons in the tests - so the pValue will always be for the variant in question with respect to the control.

(also camelCase here I think?)
```suggestion
      pValue: 0.01
```